### ISSUE TITLE
Want ability to search MFHD notes fields

### ISSUE THREAD

**Issue by gibsonjc** [CARLI/vufind#227]
NBY - We include in our MFHDs the following tags. Since they are also used in bibliographic records, we need to be able to search both MFHDs and bibs for the best retrieval of the information they contain:

506 - Restrictions on Access Note (Repeatable)
538 - System Details Note (Repeatable)
541 - Immediate Source of Acquisition Note (Repeatable)
561 - Ownership and Custodial History (Repeatable)
562 - Copy and Version Identification Note (Repeatable)
563 - Binding Information (Repeatable)
583 - Action Note (Repeatable)
843 - Reproduction Note (Repeatable)

Staff and patrons can use MFHD information by itself or in a boolean search with bibliographic records to locate groups of related materials.
Unfortunately the full effectiveness of performing these searches has been severely diminished by the presence of the static url in the MFHD for our Aeon request system.
Still, a phrase "Forms part of the Everett D. Graff Collection of Western Americana" in MFHDS retrieves over 700 records; as a booleaan OR search with bibliographic records the result is over 5400.
The phrase "former owner" appears in over 600 MFHDs; as a boolean OR search with bibliographic records the result is almost 10,000

I even a research ask me recently for a way to locate oversize materials in the collection. The most reliable method is to search MFHDs for the 852 $k prefix "oversize" There are over 10,000

In short, we have been able to use this data to be be more complete and more creative in our search strategies

**Comment by gibsonjc** [comment]
@cedelis Do we have the capability to index MFHD data? If not currently, would it be something we could try to set up?  (Demian mentioned in response to me on the email list back in Oct. that if we needed any help indexing bibs + MFHDs to let him know and he will be happy to share details.)

**Comment by cedelis** [comment]
@gibsonjc Yes, we should be able to index MFHD data. The fields get attached to the Solr record (which is mostly bib data). This is how we added call number searching capability.

**Comment by nmswanson** [comment]
Issue #363 also includes the request to index the 852 field.
